packages/react-sdk/README.md

```diff
@@ -28,17 +28,11 @@ These ready-made components can help you quickly build a chat app.
 
 - Node 16.10+
 - React 16.14+
+- Yarn v3+. See [Yarn Installation](https://yarnpkg.com/getting-started/install).
```

Comment:
this is not a requirement for use of the SDK outside of this repository. sorry for the confusion here.

Reply:
Oh my gosh, no worries at all Ry - I am sure I made some assumptions and miscommunicated in our convo - thank you for the clarification!